package.json

```diff
@@ -3,8 +3,9 @@
   "scripts": {
     "start": "node app.js",
     "dev": "hof-build watch",
-    "test": "npm run test:lint",
+    "test": "npm run test:lint && npm run test:unit",
     "test:lint": "eslint .",
+    "test:unit": "mocha --require test/test-helper.js",
```

Comment:
question: what will this do? can't you just do `_mocha` that's what I've done and runs the tests

Reply:
I think ```_mocha``` and ```mocha``` are equivalent. 

By explicitly requiring the test-helper, we always ensure it is read by mocha first regardless of test directory structure and naming conventions. Helpful if/when we add more tests files going forward I think. It can be moved to a ```mocha.opts``` file if you prefer?